oauth/oidc.py

```
@@ -467,7 +482,7 @@ def _load_keys_from_url(url, verify=True):
     """
 
     keys = []
-    r = request("GET", url, allow_redirects=True, verify=verify)
+    r = request("GET", url, allow_redirects=True, verify=verify, headers=headers)
```

Ruff (0.15.4)

[error] Probable use of `requests` call without timeout

(S113)